### DISCUSSION TITLE
Location of ini file

### DISCUSSION THREAD

**Discussion by sophof** [complexlogic/flex-launcher#66] [General]
I feel silly for asking this, but I can't find it anywhere in the documentation. Where do I need to put my own ini file on linux for it to be automatically loaded? I found the example in usr/share/flex-launcher, but I don't want to overwrite that one. I can het my own file loaded by putting it in thhe working directory, but I assume there's some default places the program looks?

**Comment by complexlogic** [comment]
`~/.config/flex-launcher`

**Reply by sophof** [reply]
Cheers!
